src/core_plugins/kibana/public/home/components/tutorial/introduction.js

```diff
@@ -21,6 +25,22 @@ export function Introduction({ description, previewUrl, title }) {
       />
     );
   }
+  let exportedFields;
+  if (exportedFieldsUrl) {
+    exportedFields = (
+      <div>
+        <EuiSpacer />
+        <KuiLinkButton
+          buttonType="secondary"
+          href={exportedFieldsUrl}
+          target="_blank"
```

Comment:
It'd be nice if we could automatically add the `rel="noopned noreferrer"` when the target is `_blank` or add a prop to add both of these for us.

Reply:
any ideas how?

Reply:
We'd have to modify the `KuiLinkButton` to do so, or add it to the `EuiLinkButton` one it's created, but you could just check the `target` prop to see if it's equal to `_blank` and add those `rels` if it is.

Reply:
I updated the EUILinkButton issue with the feature request - https://github.com/elastic/eui/issues/24#issuecomment-357258334